tangelo/molecule_library.py

```diff
@@ -16,7 +16,9 @@
 
 
 from tangelo import Molecule, SecondQuantizedMolecule
+from tangelo.helpers.utils import is_package_installed
```

Comment:
Why did you choose to change this file ?

If the user does not have pyscf installed then imports will fail without your changes.
After your changes, imports do not fail but all these molecules are None and none of the code that depends on them will work, and things will fail, I think.

Can you explain what has been gained or resolved with this ?

Rather than having imports resolving to None, i'd rather you only try to define all these molecules if pyscf has been installed and don't do it otherwise (e.g if pyscf is installed, defined all these molecule: 1 conditional branch)

I think we can define the geometries regardless, in all cases.

Reply:
Users can now follow the preference order of `SecondQuantizedMolecule`.

If pyscf is installed. Everything will work.
If only psi4 is installed. Everything except for UHF will work. It would throw an error if you tried to open molecule_library with psi4. with this modification it works.
If pyscf or psi4 are not installed. It will not work